MCSync/FancyUX.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using System.Windows.Forms;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.ComponentModel;
+
+namespace MCSync
+{
+    public class FancyButton : Button
```

Comment:
We should name the file the same as the class

Reply:
Fixed